### ISSUE TITLE
Navigation grouping.

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#487]
# Templates

Visibility: 2
Evaluation: Partially Supports
Success Criteria: 1.3.1: Info and Relationships
Needs design: 

### Description
Breadcrumbs should be in a `<nav>` element with aria-label=”breadcrumb”. On the Template pages, they are contained in an `<h1>`.

![](https://dl.airtable.com/.attachments/71b68ea4e28c4a077a94ac85726f4e76/f57d5f46/Capturedecranle2022-03-15a09_47_10.png)

**Comment by amazingphilippe** [comment]
Related finding from testing:

https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recAYwlRcb3Q7daMG?blocks=hide

> GC Notify was difficult to navigate for participants. For a user using a screenreader, the labels weren't descriptive enough (e.g. navigation regions, homepage link). For a screen magnification user, the grey banner on the top nav was too light (but other screen magnification users didn't comment on it)
